src/validator/SequentialValidator.cpp

```diff
@@ -31,16 +31,6 @@ Status SequentialValidator::validateImpl() {
     }
 
     DCHECK(!sentences.empty());
-    auto firstSentence = getFirstSentence(sentences.front());
-    switch (firstSentence->kind()) {
-        case Sentence::Kind::kLimit:
-        case Sentence::Kind::kOrderBy:
```

Comment:
How about the order by and group by?

Reply:
Should we restrict `group by` and `order by` to be used only after `PIPE`? 
Currently, `group by` and `order by` *could* be used after `semicolon` because they can refer to a variable name:
`$var = xxx; group by $var.prop`
`$var = xxx; order by $var.prop`
but in the fact they are not usable:
```
(root@nebula) [nba]> $var=go from "Tony Parker" over like yield like._dst as id; order by $var.id
[ERROR (-1009)]: SemanticError: Order by with invalid expression `$var.id'

Thu, 08 Jul 2021 14:17:15 CST

(root@nebula) [nba]> $var=go from "Tony Parker" over like yield like._dst as id; group by $var.id yield $var.id
will cause crash
```
We have to discuss on the usage of `$var`

Reply:
We could add tests firstly.

Reply:
I think the dangle limit sentence should be disabled, but the group by and order by, we should check the input/variable property expression, and set the input by this.

Reply:
ok